tests/e2e/translations/message-format.spec.js

```diff
@@ -1,51 +0,0 @@
-const utils = require('../../utils');
-const helper = require('../../helper');
-const commonElements = require('../../page-objects/common/common.po');
-const contactElements = require('../../page-objects/contacts/contacts.po');
-
-describe('MessageFormat', () => {
```

Comment:
I'm wondering if we'd keep the git history for these files if we rename them, like:
`tests/e2e/translations/message-format.wdio-spec.js` and make the selector be sensitive to the suffix, and only make the necessary changes in the files instead of rewriting the whole git history. 
What do you think?

Reply:
I think renaming the specs is a good idea so the history moves. 

I don't believe that we can get away with just updating in files as needed. I believe we are going to have to maintain two page objects as the protractor style will fail when WDIO loads the spec files. 

I'm also hoping we can get rid of code as so far WDIO does a few things better. So far implicit waiting seems to be better.  Still a little new to me but so far it seems better.

Reply:
> I believe we are going to have to maintain two page objects as the protractor style will fail when WDIO loads the spec files.

Yea, my point was to keep the files in the same location, rename them to add some suffix and make wdio/protractor include or exclude them based on this suffix. So, protractor won't load wdio files and vice versa.